docs/configuration.md

```diff
@@ -0,0 +1,23 @@
+# TesseraCT Configuration
+
+## Flags
+
+### Publication Awaiter
+
+The `enable_publication_awaiter` flag enables the publication awaiter, which waits for the checkpoint to be published before returning. As the default checkpoint interval is 10 seconds, it is recommended to update the `checkpoint_interval` flag to a lower value for performance. Note that the [maximum rate of writes to the checkpoint file is one write per second on Google Cloud Storage](https://cloud.google.com/storage/quotas#objects). Tessera enforces the minimum permitted checkpoint interval check during the TesseraCT application initialization process.
```

Comment:
Let's set a default value that works well for CT directly.

Reply:
Sounds good. I would recommend taking the minimum possible values (i.e. 1000ms for AWS and 1200ms for GCP) as the default values. What do you think?

Reply:
Let me create a separate pull request to update the default checkpoint interval value to 1500ms.